src/github.com/getlantern/peerscanner/group.go

```diff
@@ -39,13 +39,13 @@ func (g *group) deregister(h *host) {
 
 	log.Debugf("Deregistering from %v: %v", g.subdomain, h)
 
+	g.existing = nil
```

Comment:
@myleshorton I also realized that in theory, if DestroyRecord indicates a failure but actually did destroy the record, we would end up with a problem when re-registering this group since we'd still see isProxying = true and thus skip the registration.  So, I've moved the clearing of these variables before the error/return.